Mathlib/Combinatorics/SimpleGraph/Turan.lean

```diff
@@ -91,4 +110,174 @@ theorem not_cliqueFree_of_isTuranMaximal (hn : r ≤ Fintype.card V) (hG : G.IsT
   exact hGab <| le_sup_right.trans_eq ((hG.le_iff_eq <| h.sup_edge _ _).1 le_sup_left).symm <|
     (edge_adj ..).2 ⟨Or.inl ⟨rfl, rfl⟩, hab⟩
 
+open Classical in
+lemma exists_IsTuranMaximal : ∃ H : SimpleGraph V, H.IsTuranMaximal r := by
```

Comment:
A (slightly) better solution:
```suggestion
lemma exists_IsTuranMaximal :
    ∃ H : SimpleGraph V, ∃ _ : DecidableRel H.Adj, H.IsTuranMaximal r := by
```
This ensure that no weird instance is getting in the way when you destruct the existential in a proof